src/index.js

```diff
@@ -0,0 +1,136 @@
+import './style.css';
+
+const list = document.getElementById('listtodo');
+const add = document.getElementById('add');
+
+const completed = false;
+let index = 0;
```

Comment:
As mentioned in the requirements, The index should start at 1.
## For Example
the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).